server/routes/userRoute.js

```diff
@@ -6,16 +6,20 @@ import paramsValidate from '../middlewares/validateParams';
 
 const userRouter = new Router();
 const {
-  signUp, signin, allUserAccounts, createStaff, createAdmin, getAllUsers, changePassword
+  signUp,
+  signin,
+  allUserAccounts,
+  getAllUsers,
+  changePassword,
 } = userController;
 
 userRouter.post('/auth/signup', userValidate.client, signUp);
 
 userRouter.post('/auth/signin', userValidate.login, signin);
```

Comment:
I also think it will be best to remove the validation middleware during signin. I mentioned something related to this in the Channel yesterday. You remember?

Reply:
Yeah, I remember. I'll have to add it to our todos and ask the next person who will push to handle it. I didn't remember that when I pushed this. I only remembered the suggestions about the names of the staff and admin routes and I effected those